k8s/helm-chart/dynatrace-gcp-function/templates/active-gate-statefulset.yaml

```diff
@@ -77,6 +78,13 @@ spec:
               value: {{ template "environmentID" . }}
             - name: DT_ID_SEED_CLUSTER_NAME
               value: {{ .Values.clusterName }}
+            {{- if  or (eq .Values.useProxy "DT_ONLY") (eq .Values.useProxy "ALL") }}
+            - name: DT_INTERNAL_PROXY
+              valueFrom:
+                configMapKeyRef:
+                  name: dynatrace-gcp-function-config
+                  key: HTTP_PROXY
```

Comment:
HTTPS also needed

Reply:
Proxy used for communication with Dynatrace server

Value: percent encoded proxy URI

Example:

--internalProxy=https://username:password@hostname:port